tests/pems/mixed_chains/generate-certs.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Usage: ./generate_certs.sh [clean]
+# Generates mixed-chain certs for testing
+# Use argument "clean" to remove all generated certs
+
+# immediately bail if any command fails
+set -e
+
+# Generates certs with given algorithms and bits in $1$2/, ex. ec384/
```

Comment:
I don't think this comment is correct for the cert-gen function? It has more than 3 args?